### PR TITLE
Implement vertical mode

### PR DESCRIPTION
**Pull request by tjclawson-stripe**
## Summary

Add logic to parse `PaymentMethodLayout` configuration param for Android and iOS

## Motivation

Support for vertical mode in react native

## Testing

- [X] I tested this manually
- [ ] I added automated tests

## Documentation

Select one: 
- [X] I have added relevant documentation for my changes.
- [ ] This PR does not result in any developer-facing changes.

### CODE REVIEW

File: example/android/app/build.gradle [hunk 1]
```diff
@@ -283,6 +283,24 @@ dependencies {
         }
     }
     // END TODO
+
```
Comment (tjclawson-stripe) [inline example/android/app/build.gradle:286]: These dependencies were conflicting with versions in `react-native-screens`. Should be able to remove after updating `react-native-screens` version

Comment (jaynewstrom-stripe) [reply example/android/app/build.gradle:286]: This really feels like something we shouldn't do. Is there another way around this?

Comment (jaynewstrom-stripe) [reply example/android/app/build.gradle:286]: Can we do the `react-native-screens` update before this PR?

Comment (tjclawson-stripe) [reply example/android/app/build.gradle:286]: Yeah I'm looking into this now. Based on [this](https://github.com/software-mansion/react-native-screens) we might have to upgrade the react native version to upgrade the react-native-screens dep to a version that doesn't cause conflicts. Alternatively, I could just update stripe-android to 20.52.0 and then we can handle the dependency conflicts for the major version update

Comment (tjclawson-stripe) [reply example/android/app/build.gradle:286]: Downgraded to 20.52.0

**Review by jaynewstrom-stripe — DISMISSED** [review]

**Review by charliecruzan-stripe — APPROVED** [review]

**Pull request merged**